### PR TITLE
(#2673) Adding dev_packages to apache class. Allows use of httpd24u-d…

### PR DESCRIPTION
**Pull request by damonconway**
…evel from the IUS repo.

### CODE REVIEW

**Comment by DavidS** [comment]
Hi, this change looks good to me, but please add a note to the README reference section explaining the new parameter. Do not forget to squash your changes into a single commit: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits

**Comment by damonconway** [comment]
Ok, I've updated README.md and squashed my commits down to a single one.

Thanks,
Damon

File: README.md [hunk 1]
```diff
@@ -888,6 +888,19 @@ Configures a default virtual host when the class is declared. Valid options: Boo
...
+Configures a specific dev package to use. Valid options: String.  Default: 'OS default httpd dev package'.
+
+Example for using httpd 2.4 from the IUS yum repo:
+
+~~~ puppet
+class { 'apache':
+  apache_name  => 'httpd24u',
+  dev_packages => 'httpd24u-devel',
+}
+~~~
```
Comment (DavidS) [inline README.md]: Wouldn't you also need to include the `apache::dev` class?

**Comment by damonconway** [comment]
Yes, I've updated the README.

**Comment by DavidS** [comment]
Thanks for your contribution!

**Pull request merged**